.github/release-draft-template.yml

```diff
@@ -0,0 +1,36 @@
+name-template: 'v$RESOLVED_VERSION 🌻'
+tag-template: 'v$RESOLVED_VERSION'
+exclude-labels:
+  - 'skip-changelog'
+version-resolver:
+  minor:
+    labels:
+      - 'breaking-change'
+      - 'enhancement'
```

Comment:
Since your dashboard is not stable (`0.X.Y`), do NOT increase `X` for a new feature.

Reply:
```suggestion
```